themes/brainy/brainy.theme.sh

```diff
@@ -91,7 +91,7 @@ function ___brainy_prompt_user_info {
 	color=$_omb_prompt_bold_navy
 	if [ "${THEME_SHOW_SUDO}" == "true" ]; then
 		if [ $(sudo -n id -u 2>&1 | grep 0) ]; then
-			color=$_omb_prompt_bold_brown
+			color=$_omb_prompt_bold_purple
```

Comment:
Seems unrelated to setting the terminal title.

Reply:
As I commented on https://github.com/ohmybash/oh-my-bash/pull/161#issuecomment-1501130240, these changes are likely to be their personal settings, but not the changes intended for the PR.